### PR TITLE
lightning: make pre-check output message clearer (#30439)

### LINKED ISSUE #30395: lightning precheck message "smaller number for tikv-importer.disk-quota(8EiB)" too ambiguous when local disk space check fails
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)
Use lightning to import data where lightning local disk space is not enough.

### 2. What did you expect to see? (Required)
Precheck fail message should be clear, and be able to guide use to fix the issue

### 3. What did you see instead (Required)
1. "imprtestimate" should be separated as 2 words
2. "You need a smaller number for tikv-importer.disk-quota(8EiB) to finish imports" is confusing, user doesn't know how to fix the issue, smaller number than 8EiB is too ambiguous.

lightning_precheck_disk

### 4. What is your TiDB version? (Required)
Release Version: v5.4.0-nightly
Git Commit Hash: 732b7eb128ed0f251d9ce92509a3aaad63f02a80
Git Branch: heads/refs/tags/v5.4.0-nightly
Go Version: go1.16.4
UTC Build Time: 2021-12-02 08:08:09
Race Enabled: false

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #30439 to release-5.3
You can switch your code base to this Pull Request by using [git-extras](https://github.com/tj/git-extras):
```bash
# In tidb repo:
git pr https://github.com/pingcap/tidb/pull/30888
```

After apply modifications, you can push your change to this PR via:
```bash
git push git@github.com:ti-srebot/tidb.git pr/30888:release-5.3-eb6c89d6d404
```

---

### What problem does this PR solve?

Issue Number: close #30395

Problem Summary:

### What is changed and how it works?
- make tidb-lightning storage size check output message clearer
- Only write the critical message into error message

The check result is adjusted as follow:
```
local disk space may not enough to finish import, estimate sorted data size is 249.5GiB, but local available is 58.35GiB, please set `tikv-importer.disk-quota` to a smaller value than 58.35GiB or change `mydumper.sorted-kv-dir` to another disk with enough space to finish imports
```

Return error change:
```
tidb lightning encountered error:  tidb-lightning pre-check failed: local disk space may not enough to finish import, estimate sorted data size is 249.5GiB, but local available is 58.35GiB, please set `tikv-importer.disk-quota` to a smaller value than 58.35GiB or change `mydumper.sorted-kv-dir` to another disk with enough space to finish imports
```

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
make tidb-lightning pre-check output message clearer
```

### CODE REVIEW

**Review by kennytm — APPROVED** [review]

File: br/pkg/lightning/restore/check_info_test.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package restore
+
+import (
+	"context"
+	. "github.com/pingcap/check"
```
Comment (gozssky) [inline br/pkg/lightning/restore/check_info_test.go:19]: need fix imports

**Review by lichunzhu — APPROVED** [review]
LGTM

**Pull request merged**